db/schema.rb

```diff
@@ -50,11 +50,11 @@
     t.datetime "updated_at", null: false
     t.string "task_language"
     t.integer "childs_age"
-    t.integer "skill_id"
+    t.string "skill"
     t.integer "parent_id"
     t.integer "volunteer_id"
     t.index ["parent_id"], name: "index_tasks_on_parent_id"
-    t.index ["skill_id"], name: "index_tasks_on_skill_id"
+    t.index ["skill"], name: "index_tasks_on_skill"
```

Comment:
is this line necessary>

Reply:
yeah Ines noticed this as well. So added the deletion of this into the recent migration, when `rails db:migrate` is run it will sort itself out (as this file is just auto-generated then). So basically should be fine and be removed on yours when you migrate!